nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -240,10 +240,12 @@ def _postprocess_path(self, path, lang, append_index='auto', dest_type='page', p
                 path = ['rss']
             elif len(path) == 0 or append_index == 'always':
                 path = path + [os.path.splitext(self.site.config['INDEX_FILE'])[0]]
+            elif len(path) > 0 and append_index == 'never':
+                path[-1] = os.path.splitext(path[-1])[0]
```

Comment:
You mean like `sections.py` which had `return result + ['rss'], 'never'`, which now is `return result + ['rss.xml'], 'never'`?